staging/src/k8s.io/apimachinery/pkg/runtime/conversion.go

```diff
@@ -82,32 +84,91 @@ func Convert_Slice_string_To_int(input *[]string, out *int, s conversion.Scope)
 	return nil
 }
 
+func string_to_bool(in string) bool {
+	switch strings.ToLower(in) {
+	case "false", "0":
+		return false
+	default:
+		return true
+	}
+}
+
+// Convert_string_To_bool will convert a string parameter to boolean.
+// Only a value of "false", or a value of "0" resolve to false.
+// Any other value (including empty string) resolves to true.
+func Convert_string_To_bool(in *string, out *bool, s conversion.Scope) error {
+	if in == nil {
+		*out = false
+		return nil
+	}
+	*out = string_to_bool(*in)
+	return nil
+}
+
 // Convert_Slice_string_To_bool will convert a string parameter to boolean.
 // Only the absence of a value, a value of "false", or a value of "0" resolve to false.
```

Comment:
clarify "absence of a value" means zero-length slice, *not* zero-length string inside a slice

Reply:
done